fauna/manifest.json

```diff
@@ -0,0 +1,39 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "329e2845-de4a-4477-b5e9-c85f7c707316",
+  "app_id": "fauna",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "uninstallation": "README.md#Uninstallation",
+    "changelog": "CHANGELOG.md",
+    "description": "Import your Fauna query logs into Datadog.",
+    "title": "Fauna",
+    "media": [],
```

Comment:
Please include at least 3 images to be displayed on your tile demonstrating the integration value. Ex: graphs, dashboard, etc.